kernel/Dockerfile

```diff
@@ -98,8 +98,16 @@ RUN make defconfig && \
     cp System.map /out && \
     ([ -n "${DEBUG}" ] && cp vmlinux /out || true)
 
+# WireGuard
+RUN TAG="0.0.$(curl -sSL https://git.zx2c4.com/WireGuard/info/refs/ | sed -n 's/.*refs\/tags\/0\.0\.\([0-9]\+\)$/\1/p' | sort -nr | head -n 1)" && \
```

Comment:
Will do. Though it'll be `WIREGUARD_TAG` instead of `WIREGUARD_COMMIT`, since I sign tag objects, so then it can be explicitly verified via `git verify-tag`.